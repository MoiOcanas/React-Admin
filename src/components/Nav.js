import React from 'react';

//Router
import { Link } from 'react-router-dom';

//Styles
import '../styles/navbar.css';

const Nav = () => (
    <div>
        <ul>
            <i className="fab fa-react fa-2x" style={{ float: 'left', padding: '10px', marginLeft: '5px', marginTop: '5px'}}> </i>
            <span style={{ float: 'left', padding: '10px', marginTop: '10px'}}>React-Firebase Admin</span>
            <li><Link to={ process.env.PUBLIC_URL + "/"}>Home</Link></li>
        </ul>
    </div>
);

export default Nav;