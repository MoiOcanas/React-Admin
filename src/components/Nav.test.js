import React from 'react';
import { shallow } from 'enzyme';

//Component
import Nav from './Nav';

describe('Nav', () => {

    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Nav debug />);
      
        expect(component).toMatchSnapshot();
    })
    
    it('it should have an li', () => {
        const wrapper = shallow(<Nav />)
        expect(wrapper.find('li')).toHaveLength(1);
    })
})
