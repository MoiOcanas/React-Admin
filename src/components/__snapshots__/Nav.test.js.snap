// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
